01-introduction.Rmd

```diff
@@ -108,18 +112,34 @@ leaflet() %>%
   addAwesomeMarkers(lng = c(-3, 23), lat = c(52, 53), popup = popup)
 ```
 
-It would be difficult to produce Figure \@ref(fig:interactive) with only 4 lines of code in another language, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)), illustrating R's flexibility. 
-
+It would be difficult to produce Figure \@ref(fig:interactive) with only four lines of code in another language, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)), illustrating R's flexibility. 
+The use of R code therefore enables teaching geocomputation with reference to reproducible examples such as that provided in \@ref(fig:interactive) rather than abstract concepts.
 <!-- Other reasons for using R for geocompuation include: -->
 
-R provides a stable language that behaves consistently across multiple computers, free of charge.
-The use of R code therefore enables teaching geocomputation with reference to reproducible examples such as that provided in \@ref(fig:interactive) rather than abstract concepts.
-But before cracking-on with the action, a few introductory remarks are needed to explain the approach taken here and provide context.
+### Python, Java, C++
```

Comment:
I would also link-this new section in to a preceding sentence with a section cross-reference as follows: 

```
R is a functional language (Wickham 2014) with many ‘bridges’ that allow access to a treasure trove of GIS algorithms. It is thus ideal for quickly creating ‘geo-tools’, without needing to master low level languages such as C, FORTRAN and Java (see section \@ref(software-for-geocomputation)).
```

So it's neatly linked-up.